<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Capture</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Call</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Call</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoFrameLabelLocal">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoFrameLabelPartner">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>481</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textIP">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Type ip</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textPort">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Type port</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
